PreviewsSupport/PreviewsSupport/PreviewsSupport.swift

```diff
@@ -105,3 +105,57 @@ extension DefaultPreviewSource: MakeViewControllerProvider where A == UIViewCont
 }
 
 #endif
+
+@available(iOS 18.0, macOS 15.0, tvOS 18.0, watchOS 11.0, *)
+public struct PreviewModifierContextCache {
```

Comment:
Does this need to be public?

Reply:
Right now I am loading the data into the case from SnapshotCore: https://github.com/EmergeTools/SnapshotPreviews-iOS/pull/203/files#diff-57ad2b1c1655ee3a13c85b873055ca76dbf97b83eb5d07583353e130bf2be3daR117

But I can refactor this